lib/ApiRequestor.php

```diff
@@ -284,10 +291,10 @@ private static function _formatAppInfo($appInfo)
     {
         if (null !== $appInfo) {
             $string = $appInfo['name'];
-            if (null !== $appInfo['version']) {
+            if (\array_key_exists('version', $appInfo) && null !== $appInfo['version']) {
                 $string .= '/' . $appInfo['version'];
             }
-            if (null !== $appInfo['url']) {
+            if (\array_key_exists('version', $appInfo) && null !== $appInfo['url']) {
```

Comment:
```suggestion
            if (\array_key_exists('url', $appInfo) && null !== $appInfo['url']) {
```

Shouldn't this be `url`?